data/blog/utctf-2023/whattimeisit.md

```diff
@@ -0,0 +1,107 @@
+---
+title: UT CTF 2023 – What Time Is It?
+date: '2023-03-12'
+draft: false
+authors: ['legoclones']
+tags: ['UTCTF 2023', 'Forensics', 'Email']
+summary: 'Decode a timestamp hidden in Gmail Content-Type boundary to determine the time the email was sent.'
+canonical: 'https://justinapplegate.me/2023/utctf-whattimeisit/'
+---
+
+## What Time Is It? (Forensics, 911 Points)
+
+> Super Secure Company’s database was recently breached. One of the employees self reported a potential phishing event that could be related. Unfortunately, our Linux email server does not report receiving any emails on March 2, 2023. >
```

Comment:
Was the original challenge statement ended with a `>`?

Reply:
Looks like a mistake.